src/content/docs/troubleshooting.mdx

```diff
@@ -0,0 +1,18 @@
+---
+id: troubleshooting
+title: Troubleshooting
+---
+
+## Symbol not found on macOS Sonoma
+
+Most likely you have outdated [neovim](https://github.com/neovim/neovim), please update to atleast v9.4
+
+
+## <leader>fw is not working
```

Comment:
It looks like this needs to be escaped in some way, it's not building correctly. Maybe if you wrap it in `<leader>fw`

Reply:
Makes sense, escaped chars: https://github.com/AstroNvim/docs/pull/116/commits/d3a2e157361ea4259e66d39f4a73cb03e4f392f8